api/URL.json

```diff
@@ -0,0 +1,538 @@
+{
+  "api": {
+    "URL": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/URL",
+        "support": {
+          "chrome": [
+            {
+              "version_added": "2",
+              "alternative_name": "webkitURL",
+              "notes": "This feature is implemented under the non-standard name webkitURL."
+            },
+            {
+              "version_added": "32"
+            }
+          ],
+          "edge": {
+            "version_added": "12"
+          },
+          "firefox": [
+            {
+              "version_added": "4",
+              "version_removed": "18",
+              "notes": "From Gecko 2 (Firefox 4) to Gecko 18 included, Gecko supported this interface with the non-standard nsIDOMMozURLProperty internal type. As the only to access such an object was through window.URL, in practice, this didn't make any difference."
+            },
+            {
+              "version_added": "19"
```

Comment:
Reverse chronological please. version 19 is the most relevant version that should appear in the main cell.